.bazelrc

```diff
@@ -0,0 +1 @@
+envoy/.bazelrc
```

Comment:
Do you mean `import envoy/.bazelrc` ?

Reply:
`import envoy/.bazelrc` would also work, but link pointing to `.bazelrc` inside `envoy` submodule is used for consistency with how it's done for `tools/bazel.rc`.

I don't have any preference, really, @htuch?

Reply:
Oh, I wasn't aware of this is a symbolic link, it's GItHub's failure ;) either way works for me too.